### DISCUSSION TITLE
V3 enabled table plugins

### DISCUSSION THREAD

**Discussion by selbekk** [sanity-io/sanity#3821] [Sanity Studio Support]
Hi!

Is there any plugins available that provides a good table content type for the v3 studio?

I found [this one](https://github.com/fredjens/sanity-datatable) and [this one](https://github.com/rdunk/sanity-plugin-table), but both haven't been updated for years, so I'm assuming they won't work with Sanity Studio v3.

**Comment by davydog187** [comment]
Hello @selbekk, I'm currently working on migrating the `sanity-plugin-table` to V3, and plan to add additional features. You can watch my progress on [the fork](https://github.com/bitfo/sanity-plugin-table)

[PR migrating to Sanity V3](https://github.com/bitfo/sanity-plugin-table/pull/1)
